resources/build/test/test.sh

```diff
@@ -97,8 +103,10 @@ builder_parse_test() {
   if [[ "$expected" != "${_builder_chosen_action_targets[@]}" ]]; then
     builder_die "  Test: builder_parse $parameters action:target != \"$expected\""
   fi
-  if [[ "$expected_options" != "${_builder_chosen_options[@]}" ]]; then
-    builder_die "  Test: builder_parse $parameters, options != \"$expected\""
+  if [[ "${expected_options}" != "${_builder_chosen_options[@]}" ]]; then
+    echo "${_builder_chosen_options[@]}"
+    echo "${_builder_chosen_options[@]}"
+    builder_die "  Test: builder_parse $parameters, options != \"${expected_options}\""
```

Comment:
Caught a minor bug in this test's error message, which threw me off a bit while working on it.